src/components/flat-table/sort/sort.spec.js

```diff
@@ -72,7 +72,7 @@ describe("Sort", () => {
   });
 
   it("should not render `StyledSpaceHoldcer` if `sortTyp`e prop is provided", () => {
```

Comment:
I know you haven't touched this line but there are two spelling mistakes here. It should be `StyledSpaceHolder` and `sortType`.

Reply:
fixed